SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/Digital/PPMU.cs

```diff
@@ -171,6 +244,21 @@ public static void ForceCurrent(this DigitalSessionsBundle sessionsBundle, IDict
             });
         }
 
+        /// <summary>
+        /// Forces current on the target pin(s).
```

Comment:
Suggest changing to: "Forces current using specified pin- and site-unique ppmu settings."